packages/common/src/AppointmentUtils.test.ts

```diff
@@ -50,4 +52,9 @@ test("removeDonorFromDbAppointment", () => {
   expect(res.donorId).toEqual("");
   expect(res.bookingTime).toBeUndefined();
   expect(res.confirmationTime).toBeUndefined();
+
+  expect(res.lastChangeType).toEqual(BookingChange.CANCELLED);
+  expect(Date.now() - (res.lastChangeTime?.toMillis() || 0)).toBeLessThan(
```

Comment:
Why do you need the `|| 0`? 

Can it ever mask a real problem?

Reply:
The CI/CD cries the variable might be undefined.
It won't ever mask a real problem, since undefined will return 
(UNIX - 0 < 3_000 == false)

and will crash, as expected from an undefined lastChangeTime.

Reply:
If you want typescript to not be angry, you can add ! marking in un-undefined.

`Date.now() - res.lastChangeTime?.toMillis()!`